plugins/view-resources/src/components/filter/ObjectFilter.svelte

```diff
@@ -84,7 +87,7 @@
             _id: { $in: Array.from(targets.keys()) }
           }
     const options = clazz.sortingKey !== undefined ? { sort: { [clazz.sortingKey]: SortingOrder.Ascending } } : {}
-    objectsPromise = client.findAll(targetClass, resultQuery, options)
+    objectsPromise = client.findAll(targetClass, { ...resultQuery, ...(space ? { space } : {}) }, options)
```

Comment:
It break some logic, please don't filter by space by ref filters. You can test it on assignee filter for example

Reply:
Done